force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -252,11 +309,11 @@ public with sharing class ClassesObjectsMethods {
      */
     public static String createAndGetBookDetails() {
         // Create a Book object with title "A Brief History of Time" and author "Stephen Hawking"
-
+        Book hawkingBook = new Book ('A Brief History of Time', 'Stephen Hawking');
         // Call the getBookDetails method on the book object to get the details of the book
-
+        String details = hawkingBook.getBookDetails();
         // Return the details of the book
-        return null; // Replace null with the variable you used to store the result
+        return details; // Replace null with the variable you used to store the result
```

Comment:
This is totally fine.  But I wanted to give you an example of you you can work with your class and methods. 

Your method returns a String. And getBookDetails returns a String.

```
 public static String createAndGetBookDetails() {
        return new Book('A Brief History of Time', 'Stephen Hawking').getBookDetails();
    }
```